File: documentation/user/en/solve/examples/filtering-products-in-category/breadcrumb-product.cs
EvitaResponse<ISealedEntity> entities = evita.QueryCatalog(
	"evita",
	session => session.QuerySealedEntity(
        Query(
        	Collection("Product"),
        	FilterBy(
        		AttributeEquals("url", "/en/macbook-pro-13-2022"),
        		EntityLocaleEquals(CultureInfo.GetCultureInfo("en"))
        	),
        	Require(
        		EntityFetch(
        			AttributeContent("code", "name"),
        			ReferenceContent(
        				"categories",
        				EntityFetch(
        					AttributeContent("code", "name"),
        					HierarchyContent(
        						EntityFetch(
        							AttributeContent("code", "name", "level")
        						)
        					)
        				)
        			)
        		)
        	)
        )
	)
);

File: documentation/user/en/solve/examples/filtering-products-in-category/breadcrumb-product.java
final EvitaResponse<SealedEntity> entities = evita.queryCatalog(
	"evita",
	session -> {
		return session.querySealedEntity(
			query(
				collection("Product"),
				filterBy(
					attributeEquals("url", "/en/macbook-pro-13-2022"),
					entityLocaleEquals(Locale.forLanguageTag("en"))
				),
				require(
					entityFetch(
						attributeContent("code", "name"),
						referenceContent(
							"categories",
							entityFetch(
								attributeContent("code", "name"),
								hierarchyContent(
									entityFetch(
										attributeContent("code", "name", "level")
									)
								)
							)
						)
					)
				)
			)
		);
	}
);
